CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/reporting/PluginReport.java

```diff
@@ -47,7 +47,7 @@ public class PluginReport {
     private final long startTime;
     private long stopTime = -1;
 
-    private String message = null;
+    private final List<String> messageLog = new ArrayList<>();
```

Comment:
When changing the member variable definition, you should also be changing the getter/setting methods to apply to the new type, and appropriately named functions when accessing the type.
ie. getMessageLog(), setMessageLog(), addMessage(), getLastMessage(), etc.
Will likely need some refactoring of calls to this class.

Reply:
Have refactored code in line with comments. Note, have not implemented function setMessageLog() as the intended use of the attribute is to be an unchangeable record that is created at the start of the PluginReport and to persist for it's lifetime. the setMessageLog() would provide the possibility to override the mesageLog and contradict its purpose.